web/css/codeExplorer.css

```diff
@@ -0,0 +1,32 @@
+.code-explorer {
+  height: 100%;
+  padding: 1rem;
+  padding-left: 0;
+  font-family: sans-serif;
+  background-color: white;
```

Comment:
This looks like it will break the dark theme?

Reply:
Dark theme wasn't on my radar at all. Thanks for reminding me! Will take care of it.

Reply:
I just checked the production instance and the code explorer's styles remain intact when switching to the dark mode:

![image](https://user-images.githubusercontent.com/25308326/91654452-14597100-eaa9-11ea-8156-1a989d9af6ba.png)

Would you like me to style it accordingly?

Reply:
Since this is moving into the main window, I do think it needs to obey the dark/light theme now.  There was more flexibility when it was a popup window, but it would be strange if this one panel kept a white background even in the dark theme.

Reply:
Sure, makes sense. Will take care of it.